recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
+zlib-devel
```

Comment:
Bazel (the build system tensorflow uses) seems to need these and I don't think the conda zlib will work.  I'll double check locally.